src/containers/Tenant/Diagnostics/Overview/Overview.tsx

```diff
@@ -127,7 +143,9 @@ function Overview(props: OverviewProps) {
             [EPathType.EPathTypeExtSubDomain]: undefined,
             [EPathType.EPathTypeColumnStore]: undefined,
             [EPathType.EPathTypeColumnTable]: undefined,
-            [EPathType.EPathTypeCdcStream]: () => <CDCStreamInfo data={schemaData} />,
+            [EPathType.EPathTypeCdcStream]: () => (
+                <CDCStreamInfo data={schemaData} nestedChildren={currentSchemaNestedChildren} />
```

Comment:
I'm really concerned about these “nested children” props and functions everywhere
I can understand their purpose, but I'm not sure about this approach, and I really don't like the naming
this is just to let you know that one of the first comments about “nested something” really goes for all these nested somethings